specification/sql/resource-manager/readme.azureresourceschema.md

```diff
@@ -38,20 +39,15 @@ output-folder: $(azureresourceschema-folder)/schemas
 
 # all the input files in this apiVersion
 input-file:
-  - Microsoft.Sql/preview/2020-02-02-preview/managedDatabases.json
-  - Microsoft.Sql/preview/2020-02-02-preview/ServerAzureADOnlyAuthentications.json
-  - Microsoft.Sql/preview/2020-02-02-preview/managedInstances.json
-  - Microsoft.Sql/preview/2020-02-02-preview/importexport.json
-  - Microsoft.Sql/preview/2020-02-02-preview/ManagedInstanceAzureADOnlyAuthentications.json
-  - Microsoft.Sql/preview/2020-02-02-preview/ServerTrustGroups.json
   - ./Microsoft.Sql/preview/2020-02-02-preview/DatabaseSecurityAlertPolicies.json
```

Comment:
there is no changes for these files. I just order them as alphabetical and rename the file from abCd to AbCd format